### ISSUE TITLE
Possible collaboration with Joe91s Fun Bots mod? 

### ISSUE THREAD

**Issue by ludslr** [SassythSasqutch/vu-spterrains#2]
would be absolutely mind blowing to be able to play classic MP gamemodes on this map pack with bots!

**Comment by SassythSasqutch** [comment]
I agree! There's nothing stopping contributors from making tracks for Joe91's bots in singleplayer and coop levels with this mod - I'd love to do it myself but I don't have the time at the moment unfortunately.

At least, there shouldn't be any issues as far as I'm aware... if there are, please let me know!
